Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6-stretch
+
+WORKDIR /opt/avere-europarl
+
+RUN pip3 install Scrapy==1.6.0
+
+ADD LICENSE README.md ./
+ADD scraper ./scraper
+
+CMD cd scraper && ./main.py
```

Comment:
If you want to add a CMD here, it's simpler to use the absolute path: `CMD python3 /opt/avere.../main.py`